karton_whatvpn/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM certpl/artemis:latest
+
+RUN apk add git
+RUN pip install requests
```

Comment:
requests is already in the artemis image

Reply:
removed